examples/openvino/executor_runner/openvino_executor_runner.cpp

```diff
@@ -0,0 +1,325 @@
+/*
```

Comment:
Do you need a separate runner? can the executor_runner available not be used? cc @tarun292

Reply:
The aot_openvino_compiler.py script is based on Qualcomm examples like [mobilenet_v3.py](https://github.com/pytorch/executorch/blob/main/examples/qualcomm/scripts/mobilenet_v3.py).
The OpenVINO executor runner is inspired by [qnn_executor_runner.cpp](https://github.com/pytorch/executorch/blob/main/examples/qualcomm/executor_runner/qnn_executor_runner.cpp). It does not simply run the given model on a single input but also allows for model validation on a dataset prepared in a [specific way](https://github.com/pytorch/executorch/blob/main/examples/qualcomm/utils.py#L601-L622).

The default executor does not support inferring a model on datasets.
@kimishpatel, what do you think is the best way to enable model validation?
CC: @ynimmaga @alexsu52 @cavusmustafa @suryasidd